### DISCUSSION TITLE
Asset upload is failing

### DISCUSSION THREAD

**Discussion by vinukumar-vs** [Sunbird-Knowlg/Community#111] [Bugs]
We have defined ObjectCategory and ObjectCategoryDefination of **Asset**. 

While uploading the asset(image), I was getting the below error.

**API:** {{host}}/api/content/v3/create'
**Request:**
`
{
    "content": {
        "name": "Avatar",
        "keywords": [
            "vsv"
        ],
        "creator": "Vinu",
        "createdBy": "155ce3c5-713e-4749-bc1c-95d09c640914",
        "code": "org.ekstep0.5125635334872014",
        "mimeType": "image/png",
        "mediaType": "image",
        "contentType": "Asset",
        "osId": "org.ekstep.quiz.app",
        "language": [
            "English"
        ]
    }
}
`

**Response:**
`
{
    "id": "api.content.create",
    "ver": "3.0",
    "ts": "2023-03-21T06:28:30ZZ",
    "params": {
        "resmsgid": "092fa936-7343-485a-9db5-cd565aee57de",
        "msgid": null,
        "err": "404",
        "status": "failed",
        "errmsg": "Entry is not found in cassandra for content with identifier: obj-cat:certificate-template_asset_all {}"
    },
    "responseCode": "RESOURCE_NOT_FOUND",
    "result": {
        "messages": null
    }
}
`

**Comment by pallakartheekreddy** [comment]
@vinukumar-vs Why are you using the content create API for asset(media) creation? Please use the asset create API to create assets(media). Please refer below link for more details.
https://knowlg.sunbird.org/learn/product-and-developer-guide/content-service-2/content-api
